src/Printed/Bundle/Queue/Command/MaintenanceWaitForRunningCommand.php

```diff
@@ -95,7 +90,8 @@ public function execute(InputInterface $input, OutputInterface $output)
             //  If there are none we can exit the command.
             if ($count === 0) {
                 $output->writeln(sprintf('<info>There are no tasks in running state!</info>'));
-                return;
+
+                return static::SUCCESS;
```

Comment:
This needs changing to `return static::SUCCESS` or will potentially fail builds. This is not strictly a failure as the output message suggests, so will also need to change `<error>` to `<comment>`.